tests/yast2_gui/yast2_security.pm

```diff
@@ -50,6 +50,7 @@ sub run {
     wait_still_screen 1;
     wait_screen_change { type_string "5" };
     wait_screen_change { send_key "alt-o" };
+    wait_still_screen;
```

Comment:
default stilltime is 7s, could we narrow it a bit?

Reply:
yes I left the default to be sure, let me run another job with half ( 3 sec ? ) and see